rednotebook/util/markup.py

```diff
@@ -146,7 +146,11 @@ def get_markup_for_day(day, with_text=True, with_tags=True, categories=None, dat
 
     # Add date if it is not None and not the empty string
     if date:
-        export_string += '= %s =\n\n' % date
+        # Following anchor placeholder will be used as a target for every entry reference
+        # mentioning this entry's date. In postprocessing it will be replaced by relevant HTML tag.
+        date_anchor = 'DATE_ANCHOR_PLACEHOLDER_%s' % day.date.strftime('%Y-%m-%d')
```

Comment:
Is this placeholder removed for tex and txt exports?

Reply:
Now it is :)